### PR TITLE
[IMP] account move get journal sequence in name

### PR DESCRIPTION
**Pull request by mpanarin**
An improvement, mentioned in https://github.com/OCA/account-financial-tools/pull/441#pullrequestreview-79155830 by @gurneyalex

### CODE REVIEW

File: account_asset_management/models/account_asset_line.py [hunk 1]
```diff
@@ -179,11 +179,15 @@ def write(self, vals):
     def _setup_move_data(self, depreciation_date):
         self.ensure_one()
         asset = self.asset_id
+        journal = asset.profile_id.journal_id
+        sequence = journal.sequence_id.with_context(
+            ir_sequence_date=depreciation_date
+        )
         move_data = {
-            'name': asset.name,
+            'name': sequence.next_by_id(),
```
Comment (gurneyalex) [inline account_asset_management/models/account_asset_line.py]: @mpanarin at this stage, the move is not confirmed so we need to put name = '/' or even leave empty (and use the default which happens to be '/'

Then the code in the method post() will then get the proper sequence.

**Review by gurneyalex — CHANGES_REQUESTED** [review]

**Comment by mpanarin** [comment]
@gurneyalex fixed

**Comment by fclementic2c** [comment]
:+1: 
Mergeable ?

**Comment by gurneyalex** [comment]
@mpanarin there are some LINT errors => red CI. 

@fclementic2c this is a PR on an acsone branch, I can't merge this.

**Comment by sbidoul** [comment]
I think this naming of journal entries has always been like this (in v8).

I kind of remember customers complaining about this too. @adrienpeiffer @benwillig what was the conclusion? Can we change? Should this be optional?

**Comment by gurneyalex** [comment]
@sbidoul at least in France, having move names which are not sequential is a sure way to get busted by your finantial auditors / the administration. :-)

**Review by gurneyalex — CHANGES_REQUESTED** [review]
Please improve commit message: this is not an "IMP" it's a "FIX". Then explain that the previous code is generating account moves with the same number which is a really bad idea.

**Comment by sbidoul** [comment]
@gurneyalex not we are not going to maintain this branch as it is superseded by the refactoring in https://github.com/OCA/account-financial-tools/pull/677

**Comment by gurneyalex** [comment]
Hello @sbidoul do you know if there is an easy migration path between this branch and OCA#677 ?

**Comment by sbidoul** [comment]
@gurneyalex `-u` seems to be sufficient, according to Luc and our own tests.

**Comment by sbidoul** [comment]
I guess we can close this one.

**Pull request closed without merging**